### ISSUE TITLE
Passenger python bug

### ISSUE THREAD

**Issue by Raidm404** [phusion/passenger#2399]
HI,
 I am using passenger 6.0.12 . There is a python warning bug  with passenger python module. The imp module is depreciated. So if you use a python app with passenger your server  log  files will be filled with the following warning.

```
App 514754 output: /usr/share/passenger/helper-scripts/wsgi-loader.py:26: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
App 514754 output:   import sys, os, re, imp, threading, signal, traceback, socket, select, struct, logging, errno
```
Please some one can patch this module to use importlib instead of imp

**Comment by gabefair** [comment]
<img width="1339" alt="Screenshot 2023-10-16 at 10 39 13 PM" src="https://github.com/phusion/passenger/assets/2096785/4ccb3570-804d-46d5-9170-9ffa7d6d5336">

**Comment by gabefair** [comment]
<img width="1309" alt="image" src="https://github.com/phusion/passenger/assets/2096785/0dcd4157-e728-40b8-a937-2a82518c336b">


Python 3.12 no longer has the `imp` module

**Comment by CamJN** [comment]
@gabefair the patch to fix this is already merged, it is just waiting on a release.

**Comment by gabefair** [comment]
Duplicate of #2501  and merged with https://github.com/phusion/passenger/pull/2502
